common/key/keys.go

```diff
@@ -38,18 +37,13 @@ func (i *Identity) Address() string {
 	return i.Addr
 }
 
-// IsTLS returns true if this address is reachable over TLS.
-func (i *Identity) IsTLS() bool {
-	return i.TLS
-}
-
 func (i *Identity) String() string {
 	return fmt.Sprintf("{%s - %s}", i.Address(), i.Key.String())
 }
 
 // Hash returns the hash of the public key without signing the signature. The hash
-// is the input to the signature Scheme. It does _not_ hash the address & tls
-// field as those may need to change while the node keeps the same key.
+// is the input to the signature Scheme. It does _not_ hash the address field as
+// this may need to change while the node keeps the same key.
 func (i *Identity) Hash() []byte {
```

Comment:
hmm I'm just wondering if this will play nicely with previous group files - do we compare group file hashes anywhere?

Reply:
This function hasn't changed: it did not hash the TLS field and it still doesn't 😛